tests/test_verify_receipt.py

```diff
@@ -0,0 +1,428 @@
+"""
+Positive test cases for verify_receipt
+"""
+
+import os
+import json
+import io
+import unittest
+import shutil
+import tempfile
+from contextlib import redirect_stdout
+
+from datatrails_scitt_samples.datatrails.servicecontext import ServiceContext
+
+from datatrails_scitt_samples.datatrails.eventpreimage import get_event
+from datatrails_scitt_samples.datatrails.entryid import entryid_to_identity
+
+
+from datatrails_scitt_samples.scripts.generate_example_key import (
+    main as generate_example_key,
+)
+from datatrails_scitt_samples.scripts.create_hashed_signed_statement import (
+    main as create_hashed_signed_statement,
+)
+from datatrails_scitt_samples.scripts.register_signed_statement import (
+    main as register_signed_statement,
+)
+from datatrails_scitt_samples.scripts.verify_receipt import (
+    main as verify_receipt,
+)
+
+
+class TestVerifyReciept(unittest.TestCase):
+    """
+    Tests verification of a known receipt.
+    """
+
```

Comment:
add line:

     maxDiff = None

and failures with diff with be output in sensible format.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff